packages/trip-details/src/index.js

```diff
@@ -1,13 +1,4 @@
-import {
-  calculateFares,
-  calculatePhysicalActivity
-} from "@opentripplanner/core-utils/lib/itinerary";
-import { mergeMessages } from "@opentripplanner/core-utils/lib/messages";
-import { formatTime } from "@opentripplanner/core-utils/lib/time";
```

Comment:
I don't believe this is breaking since the `lib` folder is still created during the prepublish script. Therefore, these imports should still work. These imports of course wouldn't use the files from the esm folder, but things should just work when importing from package root.